src/com/hrabrov/introduction_to_java/algorithmization/one_dimensional_array/Task05.java

```diff
@@ -0,0 +1,21 @@
+package com.hrabrov.introduction_to_java.algorithmization.one_dimensional_array;
+
+public class Task05 {
+    /**
+     * Integers number a1, a2, ... , an are given.
+     * Print only those number which a[i] > i.
+     *
+     * @param array input array with integers number
+     * @return string with number
+     */
+
+    public static String selectElement(int[] array) {
```

Comment:
You need to return resulted values as an int array. It's up to `Runner` to decide how to output them, not this calculator method.